.circleci/config.yml

```diff
@@ -152,6 +152,22 @@ jobs:
             just simulate-council
             just prepare-json
             just simulate-council # simulate again to make sure the json is still valid
+  just_simulate_sep-metal-001-MCP-L1:
+    docker:
+      - image: us-docker.pkg.dev/oplabs-tools-artifacts/images/ci-builder:v0.35.0
+    steps:
```

Comment:
I recommend we define a top level variable similar to how the monorepo does it for managing versions of the docker image that is used, altho we don't want to make old ones fail

https://github.com/ethereum-optimism/optimism/blob/3e4430ee40ab72edb51ed11e5fa277ee7ae01746/.circleci/config.yml#L6

Don't need to address this as part of this PR

Reply:
Defined the top level variable in 5290ca0, but only using it in this task for now to avoid breaking others